src/music/Instruments.ts

```diff
@@ -0,0 +1,90 @@
+import Instrument from "./Instrument.js";
+import { Envelopes, AdsrConfig } from "./Envelopes.js";
+import { Note, Notes } from "./Notes.js";
+import { SampleData } from "./Samples.js";
+
+type OscType = typeof OscillatorNode.prototype.type;
```

Comment:
`type OscType = InstanceType<typeof OscillatorNode>["type"];` may be better here